essential-projects/messagebus_contracts
# 🔥 remove unused code

## What did you change?

I removed the Message-Class. When https://github.com/essential-projects/messagebus_http/pull/1 and https://github.com/essential-projects/messagebus/pull/2 are merged, it is no longer needed.

I also removed the `ioc_module.js`. not only is it not needed (as this is a contracts-package), but it would've wrongfully registered `undefined` as `'MetadataProvider'`. Looks like it was a copy-paste-error

## How can others test the changes?

This doesn't add any functionality, but it also should break stuff. You can check, if you still have any imports like this:
```
import {Message} from '@essential-projects/messagebus_contracts';
```

these need to go

## PR-Checklist

Please check the boxes in this list after submitting your PR:
